### ISSUE TITLE
Rework Readme and misc files

### ISSUE THREAD

**Issue by christianlupus** [nextcloud/cookbook#994]

**Comment by seyfeb** [comment]
If reworking the main `README.md` I’d suggest to add a screenshot of the app there. I always like a repo having a visual representation of what it contains.

**Comment by MarcelRobitaille** [comment]
Are there any further reworks needed for this issue? Is there anything I can do?

**Comment by christianlupus** [comment]
Yes, there is more to be done. All files related to user-interaction (`README.md`, `DEVELOPMENT.md`, `COPYING`/`LICENSE`, `translationfiles/README.md`) should be read for correctness and redundant/outdated information should be corrected/redacted.

Further, I would vote for reworking/restructuring the README. For example the FAQ, I would move to the user documentation. I am just chipping in some links here:
* [How to Write a Good README File for Your GitHub Project](https://www.freecodecamp.org/news/how-to-write-a-good-readme-file/)
* [Make a README](https://www.makeareadme.com/)
* [GitHub README Templates](https://www.readme-templates.com/)

**Comment by Benedikt81** [comment]
> Are there any further reworks needed for this issue? Is there anything I can do?

[User documentation](https://nextcloud.github.io/cookbook/user/) should include some mention of markdown styles to be used in the app.
